packages/common/src/logger.ts

```diff
@@ -136,3 +136,9 @@ export class Loggable {
 		this.#logger = logger;
 	}
 }
+
+export const logError = (err: unknown, logger: Logger): void => {
+	logger.error(
+		typeof err === "object" && err ? ("stack" in err ? err.stack : err) : err,
```

Comment:
if the type of the err is object, won't this output "false"?

Reply:
if the type of err is an object and err is not undefined
then if stack is in err, return err.stack
else return err
else return err

how does false come into this?

Reply:
```
typeof err === "object" && err ? ("stack" in err ? err.stack : err) : err
```
say err is not object, it becomes
```
false && err ? ("stack" in err ? err.stack : err) : err
```
false && <anything> is false, so
```
false
```